[googletts] Exception thrown on dispose()

OH Version: 4.0.0

Happens consistently, might be related to common TTS cache implementation in OH4?

```
2023-07-25 18:09:23.186 [ERROR] [.googletts.internal.GoogleTTSService] - bundle org.openhab.voice.googletts:4.0.0 (355)[org.openhab.voice.googletts.internal.GoogleTTSService(454)] : The dispose method has thrown an exception
java.lang.UnsupportedOperationException: null
	at java.util.Collections$UnmodifiableCollection.clear(Collections.java:1086) ~[?:?]
	at org.openhab.voice.googletts.internal.GoogleTTSService.dispose(GoogleTTSService.java:139) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.apache.felix.scr.impl.inject.methods.BaseMethod.invokeMethod(BaseMethod.java:245) ~[?:?]
	at org.apache.felix.scr.impl.inject.methods.BaseMethod.access$500(BaseMethod.java:41) ~[?:?]
	at org.apache.felix.scr.impl.inject.methods.BaseMethod$Resolved.invoke(BaseMethod.java:687) ~[?:?]
...

```

--- a/bundles/org.openhab.voice.googletts/src/main/java/org/openhab/voice/googletts/internal/GoogleTTSService.java
+++ b/bundles/org.openhab.voice.googletts/src/main/java/org/openhab/voice/googletts/internal/GoogleTTSService.java
@@ -136,8 +136,8 @@ public class GoogleTTSService extends AbstractCachedTTSService {
     @Deactivate
     protected void dispose() {
         apiImpl.dispose();
-        audioFormats.clear();
-        allVoices.clear();
+        audioFormats = new HashSet<>();
+        allVoices = new HashSet<>();
     }
 
     /**

